addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -324,6 +327,39 @@ def convert_links(self):
         res.update(super(Mailing, self - sms_mailings).convert_links())
         return res
 
+    def get_link_replacements_placeholders(self):
```

Comment:
``get_sms_link_replacements_placeholders`` ? to know it is sms-oriented (or could be text_link ...).